eng/Versions.props

```diff
@@ -16,7 +16,6 @@
   <PropertyGroup>
     <VersionPrefix>16.2.0</VersionPrefix>
     <AssemblyVersion>15.1.0.0</AssemblyVersion>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```

Comment:
This looks right according to https://github.com/dotnet/arcade/blob/c4bb6c1b962cb3ed164393dbb6cfc4aec2d98265/Documentation/CorePackages/Versioning.md#package-version

nit: Maybe make it

`<PreReleaseVersionLabel />`

so it's explicit that this is an intentional thing?  I don't have a strong preference.

Reply:
I will do it your way and add a comment.

Reply:
This is also nice because it creates a nice merge conflict.